flexmeasures/data/queries/sensors.py

```diff
@@ -65,3 +65,24 @@ def query_sensors_by_proximity(
         closest_sensor_query, account_id
     )
     return closest_sensor_query
+
+
+def query_sensors_by_search_terms(
+    query: Select,
+    search_terms: list[str] | None,
+) -> Select:
+    if search_terms is not None:
+        from flexmeasures.data.models.user import Account
```

Comment:
Why would you not import these at the top of the module?

Reply:
That is correct, I don't remember why I even added that there

Reply:
Have you seen this @nhoening